### DISCUSSION TITLE
Sanity check error on start up

### DISCUSSION THREAD

**Discussion by hugovdev** [Minestom/Minestom#1532] [Help]
![image](https://user-images.githubusercontent.com/23460704/203378536-dd079426-c318-42af-8e97-738807085318.png)
This happens on start-up. The line of the error is this:
`MinecraftServer minecraftServer = MinecraftServer.init();`

**Comment by emortaldev** [comment]
get**Requeste**ApiVersion()

**Reply by hugovdev** [reply]
I know, but that is not my code
